### ISSUE TITLE
@inlang/sdk-js does not resolve path to svelte.config.js correctly on windows

### ISSUE THREAD

**Issue by TazorDE** [inlang/inlang#821]
## Problem

The inlang sdk-js uses the import function to load `svelte.config.js` with the resulting string from path.resolve (as far as I understand the compiled code). Under windows this results in a string beginning with the pattern: `c:` and the import function expects a path beginning with `file://`. This results in not being able to start the SvelteKit development environment or to build for production. Trying to work around using WSL did not result in a satisfactory workflow result.

Recieved error:

```bash
$ npm run dev -- --open

> dev
> vite dev --open


node:internal/errors:490
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_UNSUPPORTED_ESM_URL_SCHEME]: Only URLs with a scheme in: file and data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'
    at new NodeError (node:internal/errors:399:5)
    at throwIfUnsupportedURLScheme (node:internal/modules/esm/resolve:972:11)
    at defaultResolve (node:internal/modules/esm/resolve:1051:3)
    at DefaultModuleLoader.resolve (node:internal/modules/esm/loader:305:12)
    at DefaultModuleLoader.getModuleJob (node:internal/modules/esm/loader:156:32)
    at DefaultModuleLoader.import (node:internal/modules/esm/loader:265:12)
    at importModuleDynamically (node:internal/modules/esm/translators:107:35)
    at importModuleDynamicallyCallback (node:internal/modules/esm/utils:87:14)
    at file:///C:/Users/Teichertm/GitRepos/WebappTemplate/node_modules/@inlang/sdk-js/dist/adapter-sveltekit/index.js:170:1572 {
  code: 'ERR_UNSUPPORTED_ESM_URL_SCHEME'
}

Node.js v20.2.0
```

### Expected behavior
The config is imported correctly and the development environment is started when running `npm run dev`.

## Steps to reproduce

Create a new svelte project in Windows
Install inlang sdk
Try to run the development environment

1. npm create svelte@latest .
2. npm install -D @inlang/sdk-js 
3. add inlang plugin to vite config
4. npm run dev -- --open

### Additional information (screenshots?)
```ts
// vite.config.ts
import { sveltekit } from '@sveltejs/kit/vite';
import { defineConfig } from 'vitest/config';
import inlangPlugin from "@inlang/sdk-js/adapter-sveltekit";

export default defineConfig({
	plugins: [inlangPlugin(), sveltekit()],
	test: {
		include: ['src/**/*.{test,spec}.{js,ts}']
	}
});
```
Similar issue but in vue
https://stackoverflow.com/questions/69665780/error-err-unsupported-esm-url-scheme-only-file-and-data-urls-are-supported-by
> Using different Node.js versions did not change the outcome

Extract from node_modules/@inlang/sdk-js/dist/adapter-sveltekit/index.js
```js
import ft from"node:path";
...
await import(ft.resolve(Br,"svelte.config.js"))
```

Possible solution: Use `pathToFileURL()` to wrap the `path.resolve()` function as iot is already imported next to path. This could possibly resolve the issue.

**Comment by samuelstroschein** [comment]
Hi Malte, 

Thank you for the elaborate issue report. One would think that `node:path` deals with windows/unix path differences. Apparently not. 

Ivan Hofer is on vacation for the next two weeks. I would not expect a fast resolution of this issue.

**Comment by ivanhofer** [comment]
The issue should be fixed in version `0.6.1`
